src/gsy_e/gsy_e_core/simulation/time_manager.py

```diff
@@ -172,7 +172,7 @@ def calc_resume_slot_and_count_realtime(
             sleep_time_s = config.slot_length.total_seconds() - seconds_elapsed_in_slot
             sleep(sleep_time_s)
 
-        self.slot_time_counter = int(time())
+        self.reset(not_restored_from_state=False)
```

Comment:
Sorry, this is not what I meant [here](https://github.com/gridsingularity/gsy-e/pull/1542#discussion_r991090307).
Please remove this line here and add it in the simulation.py in order to follow the single responsibility principle.